packages/dss-bridge/src/domains/arbitrum/ArbitrumDomainHost.sol

```diff
@@ -0,0 +1,303 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+/// OptimismDomainHost.sol -- DomainHost for Optimism
+
+// Copyright (C) 2022 Dai Foundation
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+pragma solidity ^0.8.14;
+
+import {DomainHost,TeleportGUID} from "../../DomainHost.sol";
+import {DomainGuest} from "../../DomainGuest.sol";
```

Comment:
Same here. I see they are used for the selectors, but shouldn't be necessary to expose them in the main file (I think).